selfdrive/car/chrysler/values.py

```diff
@@ -1,5 +1,5 @@
 # ruff: noqa: E501
```

Comment:
side note: can we remove this noqa?

Reply:
fingerprints are long, especially the CAN ones. Once the fingerprints are in their own file, we can remove them